SU-SWS/saa_victoria_subtheme
# ADAPT-1394: Set margin for h2 inside testimonial banner.

# READY FOR REVIEW

# Summary
-  Fix margin-top for h2's inside testimonial banner, this issue was only showing when there wasn't a span / small heading field used

# Review By (Date)
- 1/13 before end of sprint please

# Urgency
- medium, required before racialjustice site launch end of january

# Steps to Test
See what the issue looks like here, with the heading h2 text getting cut off on the top: https://racialjustice.sites.stanford.edu/
1. Look at this test page with my branch on it: https://saauserguide-test.sites.stanford.edu/site-building-guide/saa-components-stanford-sites/page-banners
2. See that in the last banner example, where there is not a span of text, just an h2 as the first child in the card_contents div, you can see that the issue is fixed (margin-top: 0 is set)
3. Review one line of code change.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket  https://stanfordits.atlassian.net/browse/ADAPT-1394
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
